tests/Makefile

```
@@ -0,0 +1,32 @@
+ORG=libhkl
+REPO=tests
+LOGFILE=build-log.txt
+TAG=latest
+LOG_DIR=.
+
+FULLNAME=$(ORG)/$(REPO):$(TAG)
+
+build ::
+	echo "# started: " `date --iso-8601="seconds"` | tee $(LOGFILE)
+	docker build \
+		-t $(FULLNAME)  \
+		./  \
+		2>&1 | tee -a $(LOGFILE)
+	echo "# finished: " `date --iso-8601="seconds"` | tee -a $(LOGFILE)
```

`--iso-8601` is not an option on all systems; `-I` is more widely available.

```suggestion
	echo "# finished: " `date -I"seconds"` | tee -a $(LOGFILE)
```